public/pages/DetectorsList/List/List.tsx

```diff
@@ -143,20 +200,25 @@ export const DetectorList = (props: ListProps) => {
   };
 
   const isFilterApplied = !isEmpty(state.queryParams.search);
+  
   const pagination = {
     pageIndex: state.page,
     pageSize: state.queryParams.size,
     totalItemCount: Math.min(MAX_DETECTORS, totalDetectors),
     pageSizeOptions: [5, 10, 20, 50],
   };
+  
   return (
     <EuiPage>
+      <SideBar />
       <EuiPageBody>
         <ContentPanel
-          title="Detectors"
+          title={"Detectors (" + totalDetectors + ")"}
```

Comment:
You could use string literal instead of this concatationaton.
title={`Detectors ${totalDetectors}`}